Document/0x04f-Testing-Network-Communication.md

```diff
@@ -108,14 +108,150 @@ If that's the case, you are now able to see the complete network traffic that is
 
 > Man-in-the-middle attacks work against any device and operating system as the attack is executed on OSI Layer 2 through ARP Spoofing. When you are MITM you might not be able to see clear text data, as the data in transit might be encrypted by using TLS, but it will give you valuable information about the hosts involved, the protocols used and the ports the app is communicating with.
 
-#### Span Port / Port Forwarding
+#### Simulating a Man-in-the-Middle Attack with an Access Point
 
-As an alternative to a MITM attack with bettercap, a Wifi Access Point (AP) or router can also be used instead. The setup requires access to the configuration of the AP and this should be clarified prior to the engagement. If it's possible to reconfigure you should check first if the AP supports either:
+##### Network Setup
+
+Simple way to simulate a man-in-the-middle (MITM) attack is to configure network where all packets between devices and the target network are going through your machine. In mobile world, this is achieved by placing Access Point between mobile devices and your machine. Your machine is a router and an Access Point.
+
+Following scenarios are possible:
+- use your machine built-in WiFi card as an Access Point and use your wired connection to connect to the target network,
+- use external USB WiFi card as an Access Point and user your machine built-in WiFi to connect to the target network (can be vice-versa) or
+- use external Access Point and redirect traffic to your machine.
+
+Scenarios with WiFi card Access Point require that WiFi card has AP (Access Point) capabilities. Additionally, you need to install tools and/or configure network to reflect man-in-the-middle position (see below).
+Scenario with external Access Point requires access to the configuration of the AP and you should check first if the AP supports either:
 
 - port forwarding or
 - has a span or mirror port.
 
-In both scenarios the AP needs to be configured to point to your machines IP. Tools like Wireshark can then again be used to monitor and record the traffic for further investigation.
+In both cases the AP needs to be configured to point to your machines IP. Your machine must be connected to the AP (via wired connection or Wifi) and you need to have connection to the target network (can be by the same connection as to the AP). Some additional configuration may be required on your machine to route traffic to the target network.
```

Comment:
```suggestion
In both cases the AP needs to be configured to point to your machines IP. Your machine must be connected to the AP (via wired connection or Wifi) and you need to have connection to the target network (can be the same connection as to the AP). Some additional configuration may be required on your machine to route traffic to the target network.
```

Reply:
@sushi2k could you please enhance your suggestion with..?

Your machine's IP
WiFi